src/tests/unit/tests/injected/frame-url-finder.test.ts

```diff
@@ -13,87 +13,89 @@ describe('frameUrlFinderTest', () => {
     });
 
     test('initialize', () => {
-        const mockFrameCommunicator = Mock.ofType(FrameCommunicator, MockBehavior.Strict);
-        const testSubject = new FrameUrlFinder(mockFrameCommunicator.object, null, null);
+        const frameCommunicatorMocks = Mock.ofType(FrameCommunicator, MockBehavior.Strict);
```

Comment:
Should `frameCommunicatorMocks` here be singular `frameCommunicatorMock` rather than plural? (Also line 29 below)

Reply:
Good catch. Updated.